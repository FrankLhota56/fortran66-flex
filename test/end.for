      Integer end
      end = 1
     0End
     1=
     2End
     3+ 1
      END