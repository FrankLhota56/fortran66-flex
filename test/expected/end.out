Line     1: INTEGER
Line     1: ID(end)
Line     2: EOS
Line     2: ID(end)
Line     2: '='
Line     2: INT_LIT(1)
Line     3: EOS
Line     3: ID(end)
Line     4: '='
Line     5: ID(end)
Line     6: '+'
Line     6: INT_LIT(1)
Line     7: EOS
Line     7: END
